src/test/java/de/tum/in/www1/artemis/PlagiarismCaseIntegrationTest.java

```diff
@@ -171,6 +180,51 @@ void testGetPlagiarismCaseForInstructor() throws Exception {
         assertThat(plagiarismCase).as("should get plagiarism case for instructor").isEqualTo(plagiarismCase1);
     }
 
+    @Test
+    @WithMockUser(username = "student1", roles = "USER")
+    void testGetPlagiarismCasesForExamForInstructor_forbidden_student() throws Exception {
+        request.getList("/api/courses/1/exams/1/plagiarism-cases/for-instructor", HttpStatus.FORBIDDEN, PlagiarismCase.class);
+    }
+
+    @Test
+    @WithMockUser(username = "tutor1", roles = "TA")
+    void testGetPlagiarismCasesForExamForInstructor_forbidden_tutor() throws Exception {
+        request.getList("/api/courses/1/exams/1/plagiarism-cases/for-instructor", HttpStatus.FORBIDDEN, PlagiarismCase.class);
+
+    }
+
+    @Test
+    @WithMockUser(username = "editor1", roles = "EDITOR")
+    void testGetPlagiarismCasesForExamForInstructor_forbidden_editor() throws Exception {
+        request.getList("/api/courses/1/exams/1/plagiarism-cases/for-instructor", HttpStatus.FORBIDDEN, PlagiarismCase.class);
+    }
+
+    @Test
+    @WithMockUser(username = "instructor1", roles = "INSTRUCTOR")
+    void testGetPlagiarismCasesForExamForInstructor() throws Exception {
+        Course examCourse = examTextExercise.getCourseViaExerciseGroupOrCourseMember();
+        Exam exam = examTextExercise.getExerciseGroup().getExam();
+        var plagiarismCasesResponse = request.getList("/api/courses/" + examCourse.getId() + "/exams/" + exam.getId() + "/plagiarism-cases/for-instructor", HttpStatus.OK,
+                PlagiarismCase.class);
+        assertThat(plagiarismCasesResponse).as("should get exam plagiarism cases for instructor").isEqualTo(examPlagiarismCases);
+        for (var submission : plagiarismCasesResponse.get(0).getPlagiarismSubmissions()) {
+            assertThat(submission.getPlagiarismComparison().getPlagiarismResult().getExercise()).as("should prepare plagiarism case response entity").isEqualTo(null);
```

Comment:
```suggestion
            assertThat(submission.getPlagiarismComparison().getPlagiarismResult().getExercise()).as("should remove unneeded elements from the response").isNull();
```
I think the `as()` description could be a bit more clear. Additionally, the more specific assertion could be used.

The same applies to the the other two lines below as well.